keps/sig-storage/1979-object-storage-support/kep.yaml

```diff
@@ -20,5 +20,5 @@ approvers:
   - "@xing-yang"
 editor: TBD
 creation-date: 2019-11-25
-last-updated: 2020-09-10
+last-updated: 2020-08-01
```

Comment:
status should be changed to "Alpha". Also specify milestone, etc. 
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L37
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L40